src/util/get-github-client-config.ts

```diff
@@ -34,24 +35,21 @@ const getGitHubClientConfigFromGitHubInstallationId = async (gitHubInstallationI
 
 const getGitHubClientConfigFromAppId = async (gitHubAppId: number | undefined): Promise<GitHubClientConfig> => {
 	const gitHubServerApp = gitHubAppId && await GitHubServerApp.getForGitHubServerAppId(gitHubAppId);
-	const gitHubCloudUrls = {
+	if (gitHubServerApp) {
+		return	{
+			hostname: gitHubServerApp.gitHubBaseUrl,
+			baseUrl: gitHubServerApp.gitHubBaseUrl,
+			appId: gitHubServerApp.appId,
+			privateKey: await gitHubServerApp.decrypt("privateKey")
+		};
+	}
+	// cloud config
+	return {
 		hostname: GITHUB_CLOUD_HOSTNAME,
-		baseUrl: GITHUB_CLOUD_API_BASEURL
+		baseUrl: GITHUB_CLOUD_API_BASEURL,
+		appId: parseInt(envVars.APP_ID),
+		privateKey: PrivateKey.findPrivateKey() || ""
```

Comment:
never do `|| ""` for private keys.  If private keys are not found, it should throw an error.  Also, shouldn't this be using the keylocator?